src/app-service.ts

```diff
@@ -92,6 +92,7 @@ export class AppService extends EventEmitter {
         app.get("/users/:userId", this.onGetUsers.bind(this));
         app.get("/rooms/:alias", this.onGetRoomAlias.bind(this));
         app.put("/transactions/:txnId", this.onTransaction.bind(this));
+        app.get("/healthcheck", this.onHealthCheck.bind(this));
```

Comment:
Any preference on the URL itself?

 - `/healthcheck`
 - `/health`
 - `/health-check`
 - `/health_check`
 
 Related to https://github.com/matrix-org/dendrite/issues/1429 which mentions `/health` but no strong opinion there.

Reply:
Tend to favour health myself as well.

Reply:
Updated to use `/health`